src/sxastarter/src/components/LayoutContainer.tsx

```diff
@@ -10,6 +10,9 @@ interface ComponentProps {
   params: ComponentParams;
 }
 
+type BreakpointKeys = 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
```

Comment:
Don't introduce any type for breakpoints enumeration as we want customers be able to provide their own names

Reply:
Removed!